po/es.po

```diff
@@ -55,11 +55,11 @@ msgstr "Terminar proceso"
 #: src/Widgets/Headerbar.vala:30
 #, fuzzy
```

Comment:
```suggestion
```

You need to remove the `fuzzy` tag, otherwise your translation won't be shown in the app.

Reply:
Thank you for your suggestion!